echopype/consolidate/loc_utils.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+import xarray as xr
+
+from ..echodata import EchoData
+from ..utils.log import _init_logger
+
+logger = _init_logger(__name__)
+
+
+def check_loc_vars_present_all_NaN(
+    echodata: EchoData, lat_name: str, lon_name: str, datagram_type: str
+):
+    """
+    Checks if any of the location variables missing or are all NaN. Recommends using another
+    datagram_type if the corresponding lat/lon variables are not missing and not not all
+    NaN.
```

Comment:
I think this is a good function. I could understand the code but not the docstring... There's some grammar problem and a redundant not. Could you revise it?

Reply:
Yup will do 👍